allocations/tasks.py

```diff
@@ -305,6 +308,47 @@ def _fill_charge_tmp_resource_ids():
                     charge.delete()
 
 
+def check_keycloak_consistency():
+    active_projects = {
+        alloc.project.charge_code
+        for alloc in Allocation.objects.filter(status="active")
+    }
+    inactive_projects = {
+        alloc.project.charge_code
+        for alloc in Allocation.objects.filter(~Q(status="active"))
+    }
+
+    LOG.info(f"CONSISTENCY: {len(active_projects)} active allocations")
+    LOG.info(f"CONSISTENCY: {len(inactive_projects)} inactive allocations")
+
+    keycloak_client = KeycloakClient()
+    groups = keycloak_client._project_admin()
+    groups_url = groups._client.get_full_url(
+        groups.get_path("collection", realm=keycloak_client.realm_name)
+    )
+
+    active_groups = {
+        group.get("name")
+        for group in groups._client.get(url=groups_url, has_active_allocation="true")
```

Comment:
I don't fully understand which API is being called here. To note, as per https://www.keycloak.org/docs-api/18.0/rest-api/#_groups_resource

the call to `GET /{realm}/groups` has the caveat "Only name and ids are returned.", and doesn't include attributes.
We might be able to avoid this by specifying `briefRepresentation=false` as per the above docs.

Reply:
Fixed. Turns out, the kwargs passed to that function do nothing and just silently fail :)))